pgxn/neon/pagestore_smgr.c

```diff
@@ -770,68 +762,66 @@ zenith_exists(SMgrRelation reln, ForkNumber forkNum)
 		return false;
 	}
 
-	request_lsn = zenith_get_request_lsn(&latest, reln->smgr_rnode.node, forkNum, REL_METADATA_PSEUDO_BLOCKNO);
+	request_lsn = neon_get_request_lsn(&latest, reln->smgr_rnode.node, forkNum, REL_METADATA_PSEUDO_BLOCKNO);
 	{
-		ZenithExistsRequest request = {
-			.req.tag = T_ZenithExistsRequest,
+		NeonExistsRequest request = {
+			.req.tag = T_NeonExistsRequest,
 			.req.latest = latest,
 			.req.lsn = request_lsn,
 			.rnode = reln->smgr_rnode.node,
-			.forknum = forkNum
-		};
+			.forknum = forkNum};
 
 		resp = page_server_request(&request);
 	}
 
 	switch (resp->tag)
 	{
-		case T_ZenithExistsResponse:
-			exists = ((ZenithExistsResponse *) resp)->exists;
-			break;
-
-		case T_ZenithErrorResponse:
-			ereport(ERROR,
-					(errcode(ERRCODE_IO_ERROR),
-					 errmsg("could not read relation existence of rel %u/%u/%u.%u from page server at lsn %X/%08X",
-							reln->smgr_rnode.node.spcNode,
-							reln->smgr_rnode.node.dbNode,
-							reln->smgr_rnode.node.relNode,
-							forkNum,
-							(uint32) (request_lsn >> 32), (uint32) request_lsn),
-					 errdetail("page server returned error: %s",
-							   ((ZenithErrorResponse *) resp)->message)));
-			break;
-
-		default:
-			elog(ERROR, "unexpected response from page server with tag 0x%02x", resp->tag);
+	case T_NeonExistsResponse:
+		exists = ((NeonExistsResponse *)resp)->exists;
+		break;
+
+	case T_NeonErrorResponse:
+		ereport(ERROR,
+				(errcode(ERRCODE_IO_ERROR),
+				 errmsg("could not read relation existence of rel %u/%u/%u.%u from page server at lsn %X/%08X",
+						reln->smgr_rnode.node.spcNode,
+						reln->smgr_rnode.node.dbNode,
+						reln->smgr_rnode.node.relNode,
+						forkNum,
+						(uint32)(request_lsn >> 32), (uint32)request_lsn),
+				 errdetail("page server returned error: %s",
+						   ((NeonErrorResponse *)resp)->message)));
+		break;
+
+	default:
+		elog(ERROR, "unexpected response from page server with tag 0x%02x", resp->tag);
 	}
 	pfree(resp);
 	return exists;
 }
 
 /*
- *	zenith_create() -- Create a new relation on zenithd storage
+ *	neon_create() -- Create a new relation on neond storage
```

Comment:
I think the 'd' was just a typo
```suggestion
 *	neon_create() -- Create a new relation on neon storage
```